python/examples/thrift/server.py

```diff
@@ -35,7 +35,7 @@ def hello(self, name):
 
 def run():
     dispatcher = TornadoDispatcher()
-    TChannel().host(dispatcher).listen(4040)
+    TChannel('localhost:4040').listen(dispatcher)
```

Comment:
We need to go back to `.host(..).listen()` (but without the port argument). Without the separation of "set a `RequestHandler`" and "listen for connections", you can't serve requests received via an outgoing connection.